ext/node/ops/vm_internal.rs

```diff
@@ -19,7 +20,22 @@ impl ContextifyScript {
     scope: &mut v8::HandleScope,
     source_str: v8::Local<v8::String>,
   ) -> Result<Self, AnyError> {
-    let source = v8::script_compiler::Source::new(source_str, None);
+    let resource_name = v8::undefined(scope);
+    let host_defined_options = get_host_defined_options(scope);
+    let origin = v8::ScriptOrigin::new(
```

Comment:
Can we please a new API for `v8::ScriptOrigin`? 🙏  It should accept a scope and an options struct, with only `resource_name` being required.